src/routes/safe/store/actions/fetchSafe.js

```diff
@@ -73,9 +73,9 @@ export const checkAndUpdateSafe = (safeAdd: string) => async (dispatch: ReduxDis
     gnosisSafe.getThreshold(),
   ])
   // Converts from [ { address, ownerName} ] to address array
-  const localOwners = localSafe.owners.map((localOwner) => localOwner.address)
-  const localThreshold = localSafe.threshold
-  const localNonce = localSafe.nonce
+  const localOwners = localSafe ? localSafe.owners.map((localOwner) => localOwner.address) : undefined
+  const localThreshold = localSafe ? localSafe.threshold : undefined
+  const localNonce = localSafe ? localSafe.nonce : undefined
```

Comment:
Can't wait when we migrate to typescript and we can use optional chaining :P (i know we can via babel plugin but I'm too lazy to add it)